app/reducers/quotes.ts

```diff
@@ -2,11 +2,17 @@ import { produce } from 'immer';
 import { createSelector } from 'reselect';
 import type { ID } from 'app/models';
 import { mutateReactions } from 'app/reducers/reactions';
+import type QuoteType from 'app/store/models/Quote';
 import type { ReactionsGrouped } from 'app/store/models/Reaction';
+import type { EntityReducerState } from 'app/utils/createEntityReducer';
 import createEntityReducer from 'app/utils/createEntityReducer';
 import joinReducers from 'app/utils/joinReducers';
 import { Quote } from '../actions/ActionTypes';
+import type { AnyAction } from 'redux';
 
+/**
+ * !!!! DONT USE ME
+ */
 export type QuoteEntity = {
   id: ID;
   text: string;
```

Comment:
Is there any reason why this is not deleted? Tried to search for it in the codebase but I couldn't find any usages of it.
Or if it is needed, could the comment be changed to use `@deprecated` to make it more visible in the places that is used that it shouldn't be used any more? (this would at least for me, make the type appear as strikethrough and give a warning when using it)
```typescript
/**
 * @deprecated Do not use
 */
export type QuoteEntity = {
  id: ID;
```

Reply:
hehe I didn't actually check. I just didn't want to remove it before we're sure. The idea is to leave these and go through once we have fixed up more types. Although It might be better just to remove them.

The downside is that if people don't fix them up as we go along, it'll be worse for a long time until someone fixes it.

The deprecated docstring is a lot better :upside_down_face:.

Reply:
Ahh right, sounds fair enough..
If we're keeping it it would also be nice to include the reason it shouldn't be used - and when it should be deleted. 

Right now it's not very informative so I wouldn't really know what to do with it if I found it like that in a couple of weeks.